packages/learner-access/src/components/LoadUserLearning.tsx

```diff
@@ -0,0 +1,272 @@
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className=" archive mt-2 row-end-4 text-black-light relative">
+          {item.availabilityStatus !== 'completed' && (
+            <ArchiveButton item={item} onArchiveSuccessAsync={onArchiveSuccessAsync} />
+          )}
+        </div>
+      </div>
+    );
+  };
+
+  interface ContentUiProps {
+    item: HydratedContentItem;
+    index?: number;
+  }
+
+  const ContentUi = ({ item }: ContentUiProps) => {
+    const [showContent, setShowContent] = useState<boolean>(false);
+    return (
+      <div
+        key={item.id}
+        className="border-solid p-4 text-black-light border-gray-light px-4 py-[0.5rem] even:bg-white-mid border-b last:border-b-0"
+      >
+        <div className="my-0 mx-auto max-w-full w-full">
+          <div className="grid items-center grid-cols-12 gap-4">
+            <div className="col-span-4">
+              <button
+                style={{ cursor: item.kind == 'inPersonEventCourse' ? 'default' : 'pointer' }}
+                className="btn btn--link btn--inherit-font dashboard-access-list-item-expander text-left"
+                onClick={() => setShowContent(prev => !prev)}
+              >
+                {item.kind !== 'inPersonEventCourse' &&
+                  (showContent ? <DownArrowIcon /> : <RightArrowIcon />)}
+                <span className="dashboard-access-list-item-expander__title text-gray-mid">
+                  {item.title}
+                </span>
+              </button>
+            </div>
+
+            <div className="col-span-2 text-gray-mid">
+              {item.displayDate &&
+                (item.kind === 'inPersonEventCourse' || item.kind === 'webinar') &&
+                formatTime(item.displayDate, undefined, 'MMM D, YYYY h A')}
+            </div>
+
+            <div className="col-span-3 text-gray-mid relative">
+              <strong className="">{item.contentTypeLabel}</strong>
+              {'  '}
+              {item.authors && item.authors.length > 0 && (
+                <div className="border-gray-mid border-solid border-l-2 h-3.5 inline my-0 mr-1 ml-px"></div>
+              )}
+              <span>
+                {' '}
+                {'  '} {item.authors}
+              </span>
+              <p className="catalog-list-item__source">{item.source}</p>
+            </div>
+
+            <div className="col-start-11 col-span-2 text-right">
+              <button
+                onClick={() => {
+                  window.location.href = window.location.hostname + '/learn/course/' + item.slug;
+                }}
+                className="bg-active-blue text-white rounded-sm cursor-pointer inline-block font-normal text-xs m-0 py-[0.15rem] px-4 relative text-center no-underline ease-in-out border-active-blue font-sans transition duration-200 leading-5"
+              >
+                {item.availabilityStatus == 'completed'
```

Comment:
@Jamal-majid-ti My bad not noticing this earlier -- since you're passing an item already handled by `hydrateContent`, you can just display `{item.callToAction` since all this logic is already handled there:

https://github.com/thoughtindustries/helium/blob/2f6f495301bd7c4b596a19b1d5a5112a97ce4a56/packages/content/src/utilities/hydrate-content/hydrate-content.ts#L101